test/document_fixture.h

```diff
@@ -1,13 +1,39 @@
 #ifndef INCLUDE_UTAP_DOCUMENT_FIXTURE_HPP
 #define INCLUDE_UTAP_DOCUMENT_FIXTURE_HPP
 
+#include <filesystem>
+#include <fstream>
 #include <memory>
 #include <stdexcept>
 #include <string>
 #include <cstring>
 
+#include <utap/StatementBuilder.hpp>
+#include <utap/property.h>
+#include <utap/typechecker.h>
 #include <utap/utap.h>
```

Comment:
messed up header inclusion: system headers go last
```suggestion
#include "utap/StatementBuilder.hpp"
#include "utap/property.h"
#include "utap/typechecker.h"
#include "utap/utap.h"

#include <filesystem>
#include <fstream>
#include <memory>
#include <stdexcept>
#include <string>
#include <cstring>
```